### ISSUE TITLE
Terminal.initTerminal(): The user does not meet the requirements to access device identifiers.

### ISSUE THREAD

**Issue by anonym24** [stripe/stripe-terminal-android#266]
I'm trying to setup Terminal for Android based on this doc https://stripe.com/docs/terminal/payments/setup-integration?terminal-sdk-platform=android

This doc mentions that I need only Location permission but it only needs Bluetooth permission as well.
Also Terminal init func will throw expection if Location Services and Bluetooth are not enabled on the device, doc doesn't mention anything of that.

    private fun initStripeTerminal() {
        if (!Terminal.isInitialized()) {
            Terminal.initTerminal(
                activityContext.applicationContext,
                LogLevel.VERBOSE,
                this,
                this
            )
        }
        terminal = Terminal.getInstance()

Now I did all of that, gave all permissions, enabled everything but now I get another issue:

```
class=Terminal message="localmobile module detected"
class=Terminal app_language=JAVA
class=ConnectionTokenManager message=refreshToken
getSerial: The user 11492 does not meet the requirements to access device identifiers.
java.lang.SecurityException: getSerial: The user 11492 does not meet the requirements to access device identifiers.
```

What does it mean? Do I need any other permission?

my deps:

    // Stripe Android SDK
    implementation 'com.stripe:stripe-android:20.15.0'
    implementation "com.stripe:stripeterminal-localmobile:2.14.0"
    implementation "com.stripe:stripeterminal-core:2.14.0"

also this (to make everything work...):

```
    configurations.all {
        c -> c.resolutionStrategy.eachDependency {
            DependencyResolveDetails dependency ->
                if (dependency.requested.group == 'org.bouncycastle') {
                    dependency.useTarget 'org.bouncycastle:bcprov-jdk15to18:1.68'
                }
        }
    }
    packagingOptions {
        pickFirst 'org/bouncycastle/x509/CertPathReviewerMessages.properties'
        pickFirst 'org/bouncycastle/x509/CertPathReviewerMessages_de.properties'
    }
}
```

**Comment by OlehSviderskyi** [comment]
my deps:

    // Stripe Android SDK
    implementation 'com.stripe:stripe-android:20.15.0'
    implementation "com.stripe:stripeterminal-localmobile:2.14.0"
    implementation "com.stripe:stripeterminal-core:2.14.0"

also this (to make everything work...):

    configurations.all {
        c -> c.resolutionStrategy.eachDependency {
            DependencyResolveDetails dependency ->
                if (dependency.requested.group == 'org.bouncycastle') {
                    dependency.useTarget 'org.bouncycastle:bcprov-jdk15to18:1.68'
                }
        }
    }
    packagingOptions {
        pickFirst 'org/bouncycastle/x509/CertPathReviewerMessages.properties'
        pickFirst 'org/bouncycastle/x509/CertPathReviewerMessages_de.properties'
    }
}

**Comment by chenmin-stripe** [comment]
Hi @anonym24 
This exception shouldn't affect SDK from functioning. You can ignore it for now. Meanwhile we will look into it.

**Comment by chr-stripe** [comment]
@anonym24 out of curiosity, what was the API level of the device you were testing this on?

**Comment by chr-stripe** [comment]
A fix for this will be released in version 2.17.0 of the Terminal Android SDK. It will no longer print an error to logcat when initializing the Terminal SDK.
